src/states/node.rs

```diff
@@ -615,6 +639,7 @@ impl Node {
                           self,
                           error,
                           pub_id);
+                    self.ban_client(&pub_id);
```

Comment:
Ah do we also want to do this for nodes ? This will do for crust-user::node as well i think